package.json

```diff
@@ -18,41 +18,52 @@
     }
   ],
   "scripts": {
-    "build": "bower install && gulp",
-    "jshint": "gulp jshint",
-    "jscs": "jscs gulpfile.js assets/scripts/*.js"
+    "build:production": "export SAGE_ENV=production && webpack",
+    "build": "export SAGE_ENV=development && webpack",
+    "watch": "export SAGE_ENV=development && node dev.js"
```

Comment:
I'm pretty sure none of these will work in Windows. Users will just get an error saying that it can't find the command `export`, or worse they might have an export command, but it does something completely different and yields wildly unexpected results.